frontend/src/services/campaign.service.ts

```diff
@@ -191,3 +198,11 @@ export async function exportCampaignStats(
     return campaignRecipients
   })
 }
+
+function errorHandler(e: AxiosError, defaultMsg: string): never {
+  console.error(e)
+  if (e.response && e.response.data && e.response.data.message) {
```

Comment:
I think we had this discussion before on not displaying too much information about errors to the user.

Reply:
<img width="1440" alt="Screenshot 2020-12-15 at 5 54 04 PM" src="https://user-images.githubusercontent.com/33819199/102199423-948fcf00-3efe-11eb-8885-e72acb8618f1.png">
was this expected?

Reply:
Yep. I added this because the default error message `An error occured while sending the campaign. Please try again.` is not useful if the custom domain is not configured. 

Instead, I thought that `This From Address cannot be used to send emails. Select another email address to send from, or contact us to investigate.` would be more helpful if it is an error with the custom domain. 

I think this is the same message shown in the custom domain modal. After catching and logging the specific error, this generic error is re-thrown in `verifyNotificationSettings`.

Reply:
^ shouldn't be just showing status code 400. `This From Address cannot be used...` should be shown. Let me double check this.

Reply:
Just verified on staging again. This is the error message shown. I think staging frontend wasn't running this branch yesterday.

![image](https://user-images.githubusercontent.com/3666479/102307726-03713480-3fa0-11eb-8add-8706bfb41eb7.png)